app/lib/check_answers_fields.yml

```diff
@@ -45,13 +45,16 @@ sections:
           type: boolean
         - attribute: passporting
           type: boolean
-        - attribute: dependants
-          type: boolean
-          screen: dependants
         - attribute: child_dependants
+          type: boolean
+          screen: dependant_details
+        - attribute: child_dependants_count
```

Comment:
If I say I have 2 child dependants, then go back and change my answer about child dependants from 'Yes' to 'No', the check answers screen will say "**Has child dependants** No"/"**Number of child dependants** 2" which seems wrong. I'd suggest adding a `skip_unless: child_dependants` to this attribute (and similar to the other three `_dependants_count` attributes.

Reply:
It was done for partner version, just not for client version